appveyor.yml

```diff
@@ -22,14 +22,18 @@ install:
     - "%PYTHON%\\python.exe -m venv venv"
     - venv\Scripts\activate.bat
     - pip.exe install -r requirements.txt
-    - pip.exe install pytest
     - pip.exe install vcrpy
+    - pip.exe install stestr
+    - pip.exe install jsonschema
 
 # Use py.test for output of xunit test results.
 test_script:
-    - py.test -v --tb=short --junit-xml=test_results.xml
+    - stestr run
+    - stestr last --subunit > test_results.subunit
 
 # Upload the test results to appveyor so they are shown on the "Tests" tab.
 on_finish:
     - ps: $wc = New-Object 'System.Net.WebClient'
-    - ps: $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\test_results.xml))
+# TODO(mtreinish): uncomment this when appveyor doesn't error on upload
```

Comment:
Could you link the issue in testing-cabal/subunit#33 with this `TODO`?
```suggestion
# TODO(mtreinish): uncomment this when testing-cabal/subunit#33 is fixed
```

Reply:
Sure, will do